code_of_conduct.md

```diff
@@ -0,0 +1,253 @@
+  - recommend an impartial replacement from the Active membership in order to maintain the five Member judicial body.
+- Recommended replacements must pass a unanimous vote of the remaining Standards Board members on the merit of their impartiality to both the accused and complainant parties. If the replacement vote does not pass, the member must continue to recommend replacements until one passes a unanimous vote.
+- In the special event that the VP of Standards recuses themselves from the investigation and hearing proceedings, they must select a new acting VP of Standards from the remaining elected Standards Board members to preside over the hearing in addition to recommending their impartial replacement. If no elected Standards Board members remain, then the VP of Standards shall select an acting VP of Standards from the approved Standards Board replacements.
+
+**Section 3.** Investigations
+
+- The VP of Standards has the sole discretion to initiate a formal or informal investigation to determine if the charges have merit and/or if they can be disposed of administratively by mutual consent of the parties involved.
+- Where a formal investigation is initiated, the Standards Board and VP of Standards should assemble a fact-finding team that will conduct an investigation of the allegations. The fact-finding team should be formed as soon as practicable and should take whatever steps it deems appropriate to investigate the allegations.
+- Each fact-finding team appointed by the Standards Board and VP of Standards should include, where possible, a member of the Standards Board and whatever other Members are deemed appropriate.
+- Any member of the Standards Board or VP of Standards who participates in the fact-finding team shall refrain from interactions with the accused and complainant parties outside of official Standards procedures for the duration of the investigation and, if deemed necessary, subsequent hearing in order to preserve impartiality.
+- The fact-finding team may utilize whatever reasonable investigative methods are at their disposal to investigate whether an offense has been committed. These methods include, but are not limited to:
+  - In-person or telephone interviews of witnesses or the accused
+  - Written Statements
+  - Requests for written information
+  - On-site visits
+  - Fact-finding conferences
+- The fact-finding team must present its findings within ten (10) days of the conclusion of its investigation. If the fact-finding team finds that the allegations are unfounded, it must present its report to the Standards Board and VP of Standards. If the allegations are determined to have merit, the chairman of the fact-finding team should prepare the matter to be presented to the Standards Board, VP of Standards.
+
+**Section 4.** Rights of the Accused
+
+- Any Member charged with an offense shall have the right to a hearing before the Standards Board and VP of Standards.
```

Comment:
A policy where complainant and accused are asked to agree on something sounds fishy to me (in addition to concerns expressed in Slack).

I feel that for a lot of reasons, the current policy of a very private Standards board is the most effective. The costs (what, in a proper court system, would mean due process violations) are outweighed by the organizational benefits. To be really clear, that means:

- resolving relatively minor issues without creating a huge organizational rift
- resolving major issues without improper influence from people who are, let's be real, our good friends
- fostering an environment where anyone feels able to come forward with a complaint against anyone

I have not been exposed to the current Standards process, but ways to mitigate concerns about this privacy could include:

- really scrupulous record-keeping by Standards
- (when appropriate, talking future here) supervision by LAN counsel